hack/builder/publish.sh

```diff
@@ -37,3 +37,6 @@ if [ "${KUBEVIRT_CRI}" = "podman" ]; then
 else
     ${KUBEVIRT_CRI} manifest push quay.io/kubevirt/builder:${VERSION}
 fi
+
+# Bump KUBEVIRT_BUILDER_IMAGE using ${VERSION}
+KUBEVIRT_BUILDER_IMAGE_TAG=${VERSION} make builder-bump
```

Comment:
It might be better to add this to the eventual prow job rather than here. Leave the publish script just responsible for publishing. 

A periodic prowjob could run to create a bump PR for the bump of the builder similar to how the kubevirtci bump is handled[1]

[1] https://github.com/kubevirt/project-infra/blob/aed0c1af1b377c70ab96c3d5622e6467854e6ec4/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml#L42

Reply:
@brianmcarey  Apologies for not getting around to this until now. I've dropped this and instead made the target a dependency of `bump-images` any updates get included the PRs provided by that periodic.